### PR TITLE
Refactor manage repository columns, move it from page into modal [SCI-4150]

### PR DESCRIPTION
**Pull request by mlorb**
Jira ticket: [SCI-4150](https://biosistemika.atlassian.net/browse/SCI-4150)

### What was done
_Refactored manage repository columns._

### CODE REVIEW

File: app/assets/stylesheets/repository_columns/index.scss [hunk 1]
```diff
@@ -22,5 +67,108 @@
...
+
+      span {
+        cursor: pointer;
+
+        &.disabled {
+          visibility: hidden;
+        }
+
+        &:hover {
+          color: $color-emperor;
```
[the remark this answers is not in the document]
Comment (aignatov-bio) [reply app/assets/stylesheets/repository_columns/index.scss]: volcano

**Comment by aignatov-bio** [comment]
In mock-ups it right information aligned by locks
Screenshot from 2020-01-14 13-26-01

**Comment by aignatov-bio** [comment]
If you drag item column down, it will create strange scroll effect

**Comment by aignatov-bio** [comment]
Add column button must be on left side

**Comment by aignatov-bio** [comment]
Some times page reload after delete/add columns. And sometimes column list empty.

File: app/assets/stylesheets/repository_columns/index.scss [hunk 2]
```diff
@@ -22,5 +67,108 @@
...
+    position: relative;
+
+    .modal-tooltiptext {
+      margin-left: 0;
+      z-index: 99999999;
+    }
+
+    li {
+      background: $color-white;
+      border-bottom: 1px solid $color-alto;
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/assets/stylesheets/repository_columns/index.scss]: later

File: app/assets/stylesheets/repository_columns/index.scss [hunk 3]
```diff
@@ -3,12 +3,57 @@
...
+      height: 390px;
+      padding: 15px 0;
+    }
+
+    .form-group {
+      margin-bottom: 10px;
+    }
+
+    .form-control {
+      font-size: 14px;
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/assets/stylesheets/repository_columns/index.scss]: later

File: app/assets/stylesheets/repository_columns/index.scss [hunk 4]
```diff
@@ -3,12 +3,57 @@
 @import "constants";
 
 #manage-repository-column {
-  .modal-footer {
-    text-align: center;
+  .modal-content {
+    font-size: 14px;
```
[the remark this answers is not in the document]
Comment (mlorb) [reply app/assets/stylesheets/repository_columns/index.scss]: later

**Pull request merged**